workspace/viewer/PdfJsViewer.ts

```diff
@@ -27,10 +27,32 @@ interface PDFPageViewStatic {
   update(scale: number, rotation?: number): void;
 
   updatePosition(): void;
+
+  resume(): void;
+}
+
+interface VisiblePageStatic {
```

Comment:
Can you add some docs describing the meaning of these fields?

Reply:
Done.

I've added some commentaries to all interfaces.